lib/mastodon/cli/accounts.rb

```diff
@@ -651,6 +647,14 @@ def migrate(username)
 
     private
 
+    def unordered_follows_for(account)
+      account.following.reorder(nil)
+    end
+
+    def unordered_followers_for(account)
+      account.followers.reorder(nil)
+    end
```

Comment:
I honestly don't think adding one level of indirection with dedicated methods for this is worth it, `account.following.reorder(nil)` is already descriptive enough imho.

Reply:
Fair enough, pushed again to keep a little repetition, but use the associations directly.